relay-server/src/actors/store.rs

```diff
@@ -673,8 +673,8 @@ impl KafkaMessage {
             Self::Attachment(message) => message.event_id.0,
             Self::AttachmentChunk(message) => message.event_id.0,
             Self::UserReport(message) => message.event_id.0,
-            Self::Session(message) => message.session_id,
-            Self::Metric(_message) => Uuid::nil(), // TODO(ja): Determine a partitioning key
+            Self::Session(_message) => Uuid::nil(), // Explicit random partitioning for sessions
```

Comment:
Note that this NIL is overwritten immediately after the match block.

Reply:
nit, can we note down the observed scenario that motivated this change?

Reply:
I was hoping in code it's enough to say that we explicitly want uniform distribution. The full reasoning is in the PR description, and given that it's a hypothesis I wasn't gonna put this in the code.